testplans/lotus-soup/go.mod

```diff
@@ -43,5 +43,3 @@ require (
 replace github.com/filecoin-project/filecoin-ffi => ../../extern/filecoin-ffi
 
 replace github.com/supranational/blst => ../../extern/blst
-
-replace github.com/filecoin-project/lotus => ../../
```

Comment:
I am not sure this plays well with Testground. `filecoin-ffi` and `blst` are bundled with the base image (as they don't change often), but we are not including `../../` when we ship the test plan to Testground, so I am a bit puzzled how this compiled and worked over the last few days.